### PR TITLE
Documentation and Qa Testing for Google Cloud sql.

### PR DESCRIPTION
**Pull request by maarioos1308**
## Description

Qa testing and documentation for Google Cloud sql.

**Fixes** ##6763

## Type of change

- [X] 📄 This change requires a documentation update

### What is the solution?

Done manual qa testing for Google Cloud sql and also updated the documentation of it.

## Checklist:

- [x] My code follows the style guidelines(PEP 8) of MindsDB.
- [x] I have updated the documentation, or created issues to update them.

### CODE REVIEW

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
Please move the `google-cloud-sql.mdx` file into the `data-integrations` folder.

Did you test it in MindsDB Cloud?

**Comment by maarioos1308** [comment]
Thank you for submitting the pull request. I have tested the changes in MindsDB cloud, but unfortunately, it seems that the functionality is not working as expected. After investigation, it appears to be related to the configuration of the MySQL instance in Google Cloud.

To resolve this issue, I need to add the public IP address of the machine that is connecting to the database. In a cloud environment, the IP address is likely to be different from that of a personal computer.
Do you have any idea about how i can resolve this ?

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
Here are the public IP addresses: https://docs.mindsdb.com/faqs/whitelist-ips

**Comment by maarioos1308** [comment]
@martyna-mindsdb I just tested it and it is working.I will add to documentation this as a tip.

**Comment by maarioos1308** [comment]
I just added an explanation about how to add the ip addresses of mindsdb in google cloud sql.

File: docs/data-integrations/google-cloud-sql.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    If you installed MindsDB locally via pip, you need to install all handler dependencies manually. To do so, go to the
+    handler's folder (mindsdb/integrations/handlers/postgres_handler) and run this command: `pip install -r
+    requirements.txt`.
+</Tip>
+
+## Usage
+
+In order to make use of this handler and connect to the Google Cloud SQL instance, you need to create a datasource with the following syntax:
+
+~~~~sql
```
Comment (martyna-mindsdb) [inline docs/data-integrations/google-cloud-sql.mdx:31]: Please update this code block to use ``` instead of ~~~~ (like the following code block).

**Comment by martyna-mindsdb** [comment]
Thanks @maarioos1308 - all looks good.

Please see the review comment above and push this small update here.

**Comment by maarioos1308** [comment]
I just changed it.Please let me know if you need anything else.

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
There is one more small update needed. Please see the review comment above.

**Comment by maarioos1308** [comment]
@martyna-mindsdb.
In witch comment you are refering to ?

**Comment by martyna-mindsdb** [comment]
@maarioos1308 
The last review comment.

Please go to `docs/data-integrations/google-cloud-sql.mdx` and change this one "mindsdb/integrations/handlers/postgres_handler" into this one "mindsdb/integrations/handlers/cloud_sql_handler".

**Comment by maarioos1308** [comment]
@martyna-mindsdb
I just did it

**Pull request merged**